models/twilio_message_enhanced.sql

```diff
@@ -99,13 +99,13 @@ final as (
         union_messages.price,
         union_messages.price_unit,
         union_messages.updated_at,
-        addresses.iso_country
+        addresses.iso_country 
 
     from union_messages
 
     left join incoming_phone_number
         on union_messages.phone_number = incoming_phone_number.phone_number
-        and union_messages.phone_number = incoming_phone_number.phone_number
+        and union_messages.phone_number = incoming_phone_number.phone_number 
```

Comment:
What is the purpose of this second join?

Reply:
Update-- I actually removed the address and incoming_phone_number models from this. I believe I misinterpreted the object definitions-- they are records of the _Twilio_ customer, not their customers themselves. I looked at our shared data to confirm

Reply:
It looks like this join was entirely removed. Was that intended?

Reply:
Sounds good!